test-harness.sh

```diff
@@ -23,10 +63,19 @@ if [ -d "$SDK_TESTING_HARNESS" ]; then
   ./scripts/down.sh
   popd
   rm -rf "$SDK_TESTING_HARNESS"
+  if [[ $SHUTDOWN == 1 ]]; then
+    echo "network shutdown complete."
```

Comment:
```suggestion
    echo "$THIS: network shutdown complete."
```

...for consitency's sake